imap_processing/cdf/config/imap_idex_l2b_variable_attrs.yaml

```
@@ -1,7 +1,144 @@
 int_fillval: &int_fillval -9223372036854775808
+int_maxval: &int_maxval 9223372036854775807
+double_fillval: &double_fillval -1.0E31
+
+# Label attributes
+mass_labels:
+  CATDESC: Labels for mass bins
+  FIELDNAM: Mass Bin Labels
+  VAR_TYPE: metadata
+  FORMAT: A8
+  DEPEND_1: mass_bins
+  DICT_KEY: SPASE>Support>SupportQuantity:Other
+
+charge_labels:
+  CATDESC: Labels for impact charge bins
+  FIELDNAM: Impact Charge Bin Labels
+  VAR_TYPE: metadata
+  FORMAT: A8
+  DEPEND_1: impact_charge_bins
+  DICT_KEY: SPASE>Support>SupportQuantity:Other
+
+spin_phase_labels:
+    CATDESC: Labels for spin phase bins
+    FIELDNAM: Spin Phase Bin Labels
+    VAR_TYPE: metadata
+    FORMAT: A8
+    DEPEND_1: spin_phase_bins
+    DICT_KEY: SPASE>Support>SupportQuantity:Other
+
+# Index attributes
+mass_bins:
+  CATDESC: Log-spaced mass bins
+  FIELDNAM: Mass Bins
+  UNITS: " "
+  FORMAT: I4
+  VAR_TYPE: support_data
+  SCALETYP: linear
+  LABLAXIS: Mass Bins
+  VALIDMIN: 0
+  VALIDMAX: 10
+  FILLVAL: *int_fillval
+  LABL_PTR_1: mass_labels
+  DICT_KEY: SPASE>Support>SupportQuantity:Other
+
+spin_phase_bins:
+    CATDESC: The spacecraft spin phase at the time of detection
+    VAR_NOTES: This is given in 4 bins [315-45, 45-135, 135-225, 225-315] degrees.
+    FIELDNAM: Spin Phase Bins
+    LABLAXIS: Spin Phase Bins
+    VAR_TYPE: support_data
+    SCALETYP: linear
+    FILLVAL: *int_fillval
+    FORMAT: I4
+    VALIDMIN: 0
+    VALIDMAX: 10
```

If only 4 bins do you want this to be 0-3
```suggestion
    VALIDMAX: 3
```